kickstart/PXELinux.erb

```diff
@@ -24,6 +24,8 @@ LABEL linux
     APPEND initrd=<%= @initrd %> ks=<%= foreman_url('provision')%> ks.device=bootif network ks.sendmac
     <% elsif @host.operatingsystem.name != 'Fedora' and @host.operatingsystem.major.to_i >= 7 -%>
     APPEND initrd=<%= @initrd %> ks=<%= foreman_url('provision')%> network ks.sendmac
+    <% elsif @host.operatingsystem.name =~ /.*Atomic.*/ -%>
+    APPEND initrd=<%= @host.medium.path %>isolinux/initrd.img repo=<%= @host.medium.path %> ks=<%= foreman_url('provision')%> ks.device=bootif network ks.sendmac
```

Comment:
That's something to do in Foreman core then by way of an OS family or changes to the OS classes, so it can download the right initrd to the TFTP proxy.  If we start changing templates to ignore the TFTP proxy and go straight to the installation media then we're going to have logic about the layout of installation media in two places.